### PR TITLE
Info line for Checkbox label #patch

### PR DESCRIPTION
**Pull request by reedsa**
Checkbox can now use the `info` prop to render a line of text below the label.

[MCFE-1120](https://jira.sendgrid.net/browse/MCFE-1120)

screen shot 2018-11-30 at 2 30 34 pm

## Merge Checklist

- [X] Includes link to associated Github Issue or JIRA ticket
- [X] Has descriptive PR title and body
- [X] All changes are backwards compatible
- [ ] QA'd by the SendGrid Style Guide designer
- [ ] Code reviewed by a SendGrid frontend lead
- [ ] If adding a new component, the component has been added to the exports in `src/index.ts`.

### CODE REVIEW

File: src/styles/checkbox-radio.module.scss [hunk 1]
```diff
@@ -113,6 +113,12 @@
   &:focus::before {
     border-color: $sg-blue;
   }
+
+  .input-checkbox-info {
+    display: block;
+    font-size: 11px;
+    color: #a9b5c0;
```
Comment (mmissey) [inline src/styles/checkbox-radio.module.scss]: should this be a styleguide variable?

File: src/checkbox.tsx [hunk 2]
```diff
@@ -37,6 +39,13 @@ export const Checkbox: React.SFC<CheckboxProps> = ({
         htmlFor={id}
       >
         {children ? children : label}
```
Comment (mmissey) [inline src/checkbox.tsx]: Is there no way to have children and a label at the same time?

Comment (reedsa) [reply src/checkbox.tsx]: That's a good thought, I don't know how this is used in all cases. It was like that already, but happy to consider changing it here.

How would children behave for such a component with or without a label? Seems that  checkboxes are only shown with text in styleguide.

Comment (mmissey) [reply src/checkbox.tsx]: I have no idea, just wondering. This is fine, especially if that is how it's been previously

**Comment by thinkingserious** [comment]
Hello @reedsa,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**